StudCity.Frontend/src/components/Message/index.jsx

```diff
@@ -6,7 +6,7 @@ const allContainer = "w-fit h-fit max-w-[80%] flex items-center";
 const displayText = "w-fit flex justify-between";
 const styleDateText = "text-xs text-[#6d6d6d]";
 const styleFullNameText = "text-sm font-bold"
-const avatarStyle = "w-12 h-12 mx-0"
+const avatarStyle = "w-12 h-12"
 const styleMessageText = "mt-3 w-fit rounded-tl-xl rounded-br-xl rounded-bl-2xl p-2"
 
 export default function Message({ userId, content, when, fullName, image, id }) {
```

Comment:
This must be inside a component.

@Serhiy-dotcom @Vova-Romaniuk What do you think?

Reply:
@maryanchuk04 do you mean the destructuring should be done inside of a component?

Reply:
Nope. I talk about this styles